libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -371,6 +381,7 @@ string IRGenerator::generateModifier(
 		m_context.resetLocalVariables();
 		Whiskers t(R"(
 			function <functionName>(<params>)<?+retParams> -> <retParams></+retParams> {
+				<sourceLocationComment>
 				<assignRetParams>
 				<evalArgs>
 				<body>
```

Comment:
`evalArgs` adds source location comments, so maybe "reset" to the original one here again?